ci_framework/plugins/action/ci_script.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/python
+
+# Copyright Red Hat, Inc.
+# Apache License Version 2.0 (see LICENSE
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+DOCUMENTATION = r'''
+---
+action: ci_script
+
+short_description: Runs a string given script remotly while providing logging
+
+description:
+    - Runs a string given script remotly while providing logging.
+    - Logs are written to the `output_dir`/../logs folder.
+    - If `cifmw_ci_script_debug` or `cifmw_debug` variables are set to true output and will be traced.
```

Comment:
could this line be rephrased a bit please, maybe it's just me, but i'm not sure i understand if it means:
- without any of _debug=true there will be no logs
- or if with _debug=true set there will be extra trace info in the logs